src/chrome/content/rules/Curse.xml

```diff
@@ -3,36 +3,208 @@
+	<target host="media-yttrium.cursecdn.com" />
+	<target host="media-zybez.cursecdn.com" />
+	<target host="minecraftforge.cursecdn.com" />
+	<target host="modloaders.cursecdn.com" />
+	<target host="mv-azeroth.cursecdn.com" />
+	<target host="soartex.cursecdn.com" />
 	<target host="static-azeroth.cursecdn.com"/>
-	<!--target host="static-copper.cursecdn.com" - 522-->
+	<target host="static-cerulean.cursecdn.com" />
+	<target host="static-copper.cursecdn.com" />
+	<target host="static-corp.cursecdn.com" />
 	<target host="static-curse.cursecdn.com" />
+	<target host="static-diablofans.cursecdn.com" />
+	<target host="static-dominaria.cursecdn.com" />
+	<target host="static-elerium-beta.cursecdn.com" />
+	<target host="static-elerium.cursecdn.com" />
 	<target host="static-hearth.cursecdn.com" />
+	<target host="static-mamba.cursecdn.com" />
+	<target host="static-mercury.cursecdn.com" />
+	<target host="static-minecraftforum.cursecdn.com" />
+	<target host="static-mods-curse.cursecdn.com" />
+	<target host="static-noxia.cursecdn.com" />
+	<target host="static-overpwn.cursecdn.com" />
+	<target host="static-seawolf.cursecdn.com" />
+	<target host="static-slapshot.cursecdn.com" />
+	<target host="static-spacejam.cursecdn.com" />
+	<target host="static-strawpoll.cursecdn.com" />
+	<target host="static-thelab.cursecdn.com" />
 	<target host="static-titanium.cursecdn.com" />
-	<!--target host="www.lolpro.com" - mixed content breakage -->
-	<target host="www.reignofgaming.net" />
+	<target host="static-touchdown.cursecdn.com" />
+	<target host="static-vox.cursecdn.com" />
+	<target host="static-yttrium.cursecdn.com" />
+	<target host="trinium.cursecdn.com" />
 
+	<target host="www.curseforge.com" />
+	<target host="aoc.curseforge.com" />
+	<target host="api.curseforge.com" />
+	<target host="authors.curseforge.com" />
+	<target host="cwow.curseforge.com" />
+	<target host="firefall.curseforge.com" />
+	<target host="forums.curseforge.com" />
+	<target host="gta.curseforge.com" />
+	<target host="kerbal.curseforge.com" />
+	<target host="legacy.curseforge.com" />
+	<target host="minecraft-beta.curseforge.com" />
+	<target host="minecraft.curseforge.com" />
+	<target host="rift.curseforge.com" />
+	<target host="rom.curseforge.com" />
+	<target host="sc2.curseforge.com" />
+	<target host="skyrim.curseforge.com" />
+	<target host="store.curseforge.com" />
+	<target host="support.curseforge.com" />
+	<target host="terraria.curseforge.com" />
+	<target host="teso.curseforge.com" />
+	<target host="tsw.curseforge.com" />
+	<target host="war.curseforge.com" />
+	<target host="wildstar.curseforge.com" />
+	<target host="worldoftanks.curseforge.com" />
+	<target host="wow-beta.curseforge.com" />
+	<target host="wow.curseforge.com" />
 
-	<securecookie host="^\." name="^(?:__cfduid|cf_clearance)$" />
+	<target host="curseinc.com" />
+	<target host="www.curseinc.com" />
 
+	<target host="mmo-champion.com" />
+	<target host="www.mmo-champion.com" />
```

Comment:
Dunno how you brought this back when my PR #7887 clearly excludes this since it doesn't work due to mixed content.